bemani/frontend/popn/endpoints.py

```diff
@@ -15,7 +15,7 @@
 popn_pages = Blueprint(
     'popn_pages',
     __name__,
-    url_prefix='/popn',
+    url_prefix='/pnm',
```

Comment:
This shouldn't matter, but curious why the change?

Reply:
I've change this due to how song_url is made, in triggers.py. It takes the game.value as part of the url.

Reply:
That sounds legit enough to me. Maybe we should actually make the url prefix for each game the value of the enum? Seems like good tech debt cleanup for me or somebody else at some point.